service/src/main/java/bio/terra/workspace/service/workspace/CloudSyncRoleMapping.java

```diff
@@ -14,20 +14,22 @@
  * CustomGcpIamRoleMapping}), and should be removed from this list.
  */
 public class CloudSyncRoleMapping {
-  private static final List<String> READER_PERMISSIONS = ImmutableList.of("roles/viewer");
```

Comment:
Checking my understanding here: We no longer need the generic `viewer` role for readers because they will be given read access on individual resources instead, yes?

Reply:
Yep, exactly. We no longer need project level roles for buckets/datasets/notebooks, so I replaced the broad `roles/viewer` with a much narrower set of permissions covering the pieces we still need/haven't replaced.